### PR TITLE
add r-rbiom recipe

### PR DESCRIPTION
**Pull request by dansmith01**
Generated with conda_r_skeleton_helper

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (LICENSE file included in this PR).
- [x] Source is from official source. (CRAN)
- [x] Package does not vendor other packages.
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe.
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dansmith01** [comment]
I agree to be listed as a maintainer.

**Comment by dansmith01** [comment]
@conda-forge-admin, please ping team
Ready for review!

File: recipes/r-rbiom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  
+  summary: A toolkit for working with Biological Observation Matrix ('BIOM') files. Features
+    include reading/writing all 'BIOM' formats, rarefaction, alpha diversity, beta diversity
+    (including 'UniFrac'), summarizing counts by taxonomic level, and sample subsetting.
+    Standalone functions for reading, writing, and subsetting phylogenetic trees are
+    also provided. All CPU intensive operations are encoded in C with multi-thread support.
+
+extra:
+  recipe-maintainers:
+    - dansmith01
```
Comment (cbrueffer) [inline recipes/r-rbiom/meta.yaml:80]: Please add `conda-forge/r` as maintainer.

File: recipes/r-rbiom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+test:
+  commands:
+    - $R -e "library('rbiom')"           # [not win]
+    - "\"%R%\" -e \"library('rbiom')\""  # [win]
+
+about:
+  home: https://cmmr.github.io/rbiom/index.html
+
+  license: AGPL-3.0-or-later
```
Comment (cbrueffer) [inline recipes/r-rbiom/meta.yaml:68]:
```suggestion
  license: AGPL-3.0-only
```

File: recipes/r-rbiom/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  license_family: AGPL
+  license_file: LICENSE
```
Comment (cbrueffer) [inline recipes/r-rbiom/meta.yaml:70]:
```suggestion
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/AGPL-3
```

Comment (cbrueffer) [reply recipes/r-rbiom/meta.yaml:70]: This makes it use the `r-base` installed license file; please remove the LICENSE file in this recipe.

**Review by cbrueffer — CHANGES_REQUESTED** [review]

**Review by cbrueffer — APPROVED** [review]

**Comment by cbrueffer** [comment]
Looks good now, thanks!

**Comment by dansmith01** [comment]
@cbrueffer thanks for adding!

**Pull request merged**